src/btree/bt_handle.c

```diff
@@ -181,6 +180,17 @@ __wt_btree_close(WT_SESSION_IMPL *session)
 
 	btree->bulk_load_ok = false;
 
+	/* Reset eviction information. */
+	__wt_spin_destroy(session, &btree->evict_lock);
```

Comment:
I'm confused about how this works: eviction can pick up a file that has been marked dead but still has a tree in cache.  With these changes, won't that try to acquire this spin lock and fail?